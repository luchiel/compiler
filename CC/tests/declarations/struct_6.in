struct a
{
    float *r[];
    struct { int g; };
} b[10];