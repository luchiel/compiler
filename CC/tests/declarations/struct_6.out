Error: ParseError:4:22 non-abstract declarator expected
